### PR TITLE
Support Hunyuan DIT lora

### PR DESCRIPTION
**Pull request by xuhuaren**
image
workflow:https://github.com/Tencent/HunyuanDiT/blob/main/comfyui-hydit/workflow/workflow_v1.1_lora.json
lora model: https://drive.google.com/file/d/1_2ZZSYn4vALMHnhNfjF03R_7mJ52-RZj/view?usp=drive_link

### CODE REVIEW

File: models [hunk 1]
```diff
@@ -0,0 +1 @@
+/apdcephfs_cq8/share_1367250/xuhuaren/comfyui_project/ComfyUI/models
```
Comment (mcmonkey4eva) [inline models:1]: This PR is a mess? what's all the random unrelated edits?

File: .gitignore [hunk 2]
```diff
@@ -5,7 +5,7 @@ __pycache__/
 !/input/example.png
 /models/
 /temp/
-/custom_nodes/
+#/custom_nodes/
```
Comment (mcmonkey4eva) [inline .gitignore:8]: wtf

**Comment by comfyanonymous** [comment]
Implemented in: https://github.com/comfyanonymous/ComfyUI/commit/a9f04edc5887095f312bc16d9a6617e08c764678
